### ISSUE TITLE
homebrew/versions/protobuf260 depreciated

### ISSUE THREAD

**Issue by wangxinxi** [litecoin-project/litecoin#291]
The build notes for OS X should be updated https://github.com/litecoin-project/litecoin/blob/master/doc/build-osx.md

The following commands should be used. 

```bash
brew install protobuf@2.6
sudo ln -s /usr/local/Cellar/protobuf\@2.6/2.6.0/lib/pkgconfig/protobuf.pc /usr/local/lib/pkgconfig/
sudo ln -s /usr/local/Cellar/protobuf\@2.6/2.6.0/bin/protoc /usr/local/bin/
```

**Comment by thrasher-** [comment]
Should be fixed in: https://github.com/thrasher-/litecoin/blob/0.14-dev/doc/build-osx.md

When 0.14 gets pushed, we will test it first before closing this.

**Comment by thrasher-** [comment]
https://github.com/litecoin-project/litecoin/pull/331 has been merged to 0.13, tested and working fine. 0.14 has this fixed also.
